utilities/config_to_json.py

```diff
@@ -105,11 +129,9 @@ def get_config_contents(config_file_name):
         return ''
 
 
-def get_stanzas(config_data: str):
-    """ returns a list of stanzas in a config file
+def get_stanzas(config_data: dict):
```

Comment:
I'm still pretty weak in Python... does this actually _do_ anything or is it just a helpful reminder of what is expected? (This is not a criticism, it just seems like this changed with no other code needing to change which makes it feel like like it's a potential for confusion as much as a helpful bit... but I'm curious to learn more).

Reply:
its just a reminder of what's expected. The function definitely takes a dictionary, so this is now correct. I went back to see if that function had ever taken a string, but it doesn't look like it.

Reply:
This can actually do something. It's ignored at runtime but the type hinting here could definitely be used during continuous integration for static analysis with [mypy](http://mypy-lang.org/). I've been wanting to start doing this on my projects.

Reply:
Thanks. That makes sense to me. I think my confusion / concern was if it wasn't being tested in some way it is likely to be wrong (which it was before this PR fixed it) and cause more harm than good. Thanks for sharing the tool that might be useful to prevent my concern. 🍻